test/functional/test_runner.py

```diff
@@ -157,6 +157,21 @@
 # Place EXTENDED_SCRIPTS first since it has the 3 longest running tests
 ALL_SCRIPTS = EXTENDED_SCRIPTS + BASE_SCRIPTS
 
+def check_script_prefixes():
+    EXPECTED_VIOLATION_COUNT = 76
+    LEEWAY = 10
+
+    ok = {"feature", "interface", "mempool", "mining", "p2p", "rpc", "wallet"}
+    found = set(x.split("_",1)[0] for x in ALL_SCRIPTS if x != "example_test.py")
```

Comment:
nit: this is testing the number of prefixes violating the rule rather than the number of test names violating the rule. If I had `badprefix_test1.py` and `badprefix_test2.py`, that would only count as one violation.

Reply:
fixed